src/main/java/org/dcsa/ebl/repository/ClauseRepository.java

```diff
@@ -0,0 +1,25 @@
+package org.dcsa.ebl.repository;
+
+import org.dcsa.core.repository.ExtendedRepository;
+import org.dcsa.ebl.model.Clause;
+import org.springframework.data.r2dbc.repository.Modifying;
+import org.springframework.data.r2dbc.repository.Query;
+import reactor.core.publisher.Flux;
+import reactor.core.publisher.Mono;
+
+import java.util.UUID;
+
+public interface ClauseRepository extends ExtendedRepository<Clause, UUID> {
+    @Query("SELECT id, clause_content from carrier_clauses " +
```

Comment:
```suggestion
    @Query("SELECT carrier_clauses.* from carrier_clauses " +
```

(Future proof in case more fields are added to or removed from the Clause object)